moxy-compiler/src/test/java/moxy/compiler/ViewStateNotMvpInterfaceTest.kt

```diff
@@ -0,0 +1,161 @@
+package moxy.compiler
+
+import com.tschuchort.compiletesting.KotlinCompilation
+import com.tschuchort.compiletesting.SourceFile
+import org.intellij.lang.annotations.Language
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class ViewStateNotMvpInterfaceTest : CompilerTest() {
+
+    @Test
+    fun successOtherModuleViewInterface() {
+        @Language("kotlin") val sources = """
+            import moxy.InjectViewState
+            import moxy.MvpPresenter
+            import moxy.view.TestView
+            import moxy.MvpView
+            import moxy.viewstate.strategy.AddToEndStrategy
+            import moxy.viewstate.strategy.StateStrategyType
+            import moxy.viewstate.strategy.OneExecutionStateStrategy
+            
+            interface OtherModuleView {
+                fun helloFromOtherModule()
+            }
+            
+            interface MainView : MvpView, OtherModuleView {
+                @StateStrategyType(AddToEndStrategy::class)
+                fun printLog(msg: String)
+            
+                @StateStrategyType(OneExecutionStateStrategy::class)
+                fun openKtxActivity()
+            
+                @StateStrategyType(AddToEndStrategy::class)
+                override fun helloFromOtherModule()
+            }
+
+            class MainPresenter: MvpPresenter<MainView>() {
+            
+                override fun onFirstViewAttach() {
+                    super.onFirstViewAttach()
+                    viewState.printLog("TEST")
+                }
+            
+                fun printLog() {
+                    viewState.printLog("TEST print log ${hashCode()}")
+                }
+            
+                fun onOpenKtxButtonClick() {
+                    viewState.openKtxActivity()
+                }
+            }
```

Comment:
It is not necessary to create such long presenters in tests. You can omit body of presenter altogether, leaving only constructor call